src/main/java/com/enterprise/fundraisermanager/controller/FundraiserManagerController.java

```
@@ -1,97 +1,125 @@
+ * @author .
+ */
+@Controller
+public class FundraiserManagerController {
+    @Autowired
+    IFundraiserService fundraiserService;
+
+    /**
+     * Handle the root (/) endpoint and return a start page.
+     * @return start page
+     */
+    @RequestMapping("/")
+    public String index(Model model){
+        model.addAttribute("Fundraiser", fundraiserService.fetchAll());
+        return "start";
+    }
+
+    /**
+     * Saves fundraiser to database.
+     * @param fundraiser
+     * @return start page
+     */
+    @RequestMapping("/saveFundraiser")
+    public String saveFundraiser(Fundraiser fundraiser){
+        try {
+            fundraiserService.save(fundraiser);
+        } catch (Exception e) {
+            e.printStackTrace();
+            return "start";
+        }
+        return "start";
+    }
+
+    /**
+     * Get all information regarding the fundraiser
+     * @param id
+     * @return all information regarding the fundraiser
+     */
+    @GetMapping("/fundraiser/{id}/")
+    public ResponseEntity fetchFundraiserByID(@PathVariable("id") Integer id) {
+        Fundraiser foundFundraiser = fundraiserService.fetchById(id);
+        HttpHeaders headers = new HttpHeaders();
+        headers.setContentType(MediaType.APPLICATION_JSON);
+        return new ResponseEntity(foundFundraiser, headers, HttpStatus.OK);
+    }
+
+    /**
+     * Create a new fundraiser
+     * @param fundraiser
+     * @return new fundraiser object
+     */
+    @PostMapping(value="/fundraiser", consumes="application/json", produces = "application/json")
+    public Fundraiser createFundraiser(@RequestBody Fundraiser fundraiser) {
+        Fundraiser newFundraiser = null;
+        try {
+            newFundraiser = fundraiserService.save(fundraiser);
+        }catch (Exception e) {
+        }
```

Empty catch block is a no-no.